### ISSUE TITLE
Example code is not tested or run

### ISSUE THREAD

**Issue by LilithHafner** [Clemapfel/Mousetrap.jl#66]
This leads to issues like
https://github.com/Clemapfel/Mousetrap.jl/pull/65
https://github.com/Clemapfel/Mousetrap.jl/pull/62
https://github.com/Clemapfel/Mousetrap.jl/pull/56

**Comment by Clemapfel** [comment]
I agree, the main issue is that Documenter.jl doctest macro does not work with Mousetrap, as it needs to create a new window and application instance anytime any of the code snippets is run, since they are all completely sandboxed.

You are still right though, I should find another automated system to run all the example snippets

**Comment by Clemapfel** [comment]
Fixed by multiple commits such as https://github.com/Clemapfel/Mousetrap.jl/commit/b72910a73078873423d68faeea98ac52041a7aa4, all runnable code in the docs is now run once when the docs are built, avoiding future typos. 

This fix will go live with 3.4

**Comment by LilithHafner** [comment]
Thank you! Well done! Yay!
